dev-client/src/components/links/InternalLink.tsx

```diff
@@ -30,22 +30,25 @@ type InternalLinkProps = {
 };
 
 export const InternalLink = ({label, onPress, url}: InternalLinkProps) => {
+  const [pressed, setPressed] = useState(false);
```

Comment:
I cleaned up some of `InternalLink` to make its implementation more consistent with the updated `ExternalLink`.